schema.sql

```diff
@@ -312,7 +312,7 @@ CREATE TABLE IF NOT EXISTS `player_spells` (
 CREATE TABLE IF NOT EXISTS `player_storage` (
   `player_id` int(11) NOT NULL DEFAULT '0',
   `key` int(10) unsigned NOT NULL DEFAULT '0',
-  `value` int(11) NOT NULL DEFAULT '0',
+  `value` bigint(20) NOT NULL DEFAULT '0',
```

Comment:
You make it inconsistent by having a schema snapshot with latest change but without actually upgrading the version number in the schema. You don't really need to edit this file.